src/Mapbender/RoutingBundle/Element/Routing.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Mapbender\RoutingBundle\Element;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpFoundation\Request;
+use Exception;
+use Mapbender\Component\Element\AbstractElementService;
+use Mapbender\Component\Element\TemplateView;
+use Mapbender\CoreBundle\Entity\Element;
+use Mapbender\RoutingBundle\Component\RoutingHandler;
+use Mapbender\RoutingBundle\Component\SearchHandler;
+use Mapbender\RoutingBundle\Component\ReverseGeocodingHandler;
+
+/**
+ * Class RoutingElement
+ * @package Mapbender\RoutingBundle\Element
+ * @author Christian Kuntzsch <christian.kuntzsch@wheregroup.com>
+ * @author Robert Klemm <robert.klemm@wheregroup.com>
+ */
+class Routing extends AbstractElementService
+{
+    protected RoutingHandler $routingHandler;
+
+    protected SearchHandler $searchHandler;
+
+    protected ReverseGeocodingHandler $reverseGeocodingHandler;
+
+    public function __construct(RoutingHandler $routingHandler, SearchHandler $searchHandler, ReverseGeocodingHandler $reverseGeocodingHandler) {
+        $this->routingHandler = $routingHandler;
+        $this->searchHandler = $searchHandler;
+        $this->reverseGeocodingHandler = $reverseGeocodingHandler;
+    }
+
+    /**
+     * @return string
+     */
+    public static function getClassTitle() : string
+    {
+        return 'mb.routing.backend.title';
+    }
+
+    /**
+     * @return string
+     */
+    public static function getClassDescription() : string
+    {
+        return 'mb.routing.backend.description';
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getRequiredAssets(Element $element)
+    {
+        return [
+            'js' => [
+                '@MapbenderRoutingBundle/Resources/public/mapbender.element.routing.js',
+            ],
+            'css' => [
+                '@MapbenderRoutingBundle/Resources/public/css/jquery-ui.css',
+                '@MapbenderRoutingBundle/Resources/public/sass/routing-icons.scss',
+                '@MapbenderRoutingBundle/Resources/public/sass/mapbender.element.routing.scss',
+            ],
+            'trans' => [
+                'mb.routing.*',
+            ],
+        ];
+    }
+
+    public function getWidgetName( Element $element ): string
+    {
+        return 'mapbender.mbRouting';
+    }
+
+    /**
+     * @return array
+     */
+    public static function getDefaultConfiguration()
+    {
+        return [
+            'title' => 'Routing',
+            'advancedSettings' => false,
+            'autoSubmit' => false,
+            'allowIntermediatePoints' => false,
+            'useSearch' => false,
+            'useReverseGeocoding' => false,
+            'buffer' => 0,
+            'infoText' => '{start} → {destination} </br> {length} will take {time}',
```

Comment:
is it possible to translate the default here? So someone with a german machine gets a german default text when first creating the element?